hail/python/hailtop/batch/docs/service.rst

```diff
@@ -55,29 +55,41 @@ Billing
 
 The cost for executing a job depends on the underlying machine type and how much CPU and
 memory is being requested. Currently, Batch runs all jobs on 16 core, preemptible, n1-standard
-machines with 100 GB of disk total. The costs are as follows:
+machines with 20 GB of persistent SSD boot disk and 375 GB of local SSD. The costs are as follows:
 
 - Compute cost
    = $0.01 per core per hour
 
 - Disk cost
-   .. code-block:: text
 
-       Average number of days per month = 365.25 / 12 = 30.4375
+  Average number of days per month = 365.25 / 12 = 30.4375
 
-       Cost per GB per month = $0.17
+  - Boot Disk
 
-       Cost per core per hour = $0.17 * 100 / 30.4375 / 24 / 16
+    .. code-block:: text
 
-   = $0.001454 per core per hour
+        Cost per GB per month = $0.17
+
+        Cost per core per hour = $0.17 * 20 / 30.4375 / 24 / 16
+
+  - Local SSD
+
+    .. code-block:: text
+
+        Cost per GB per month = $0.048
+
+        Cost per core per hour = $0.048 * 375 / 30.4375 / 24 / 16
+
+
+   = $0.00183 per core per hour
```

Comment:
this gets displayed as a block quote, maybe the equals sign needs to be one indent further to the left?

Reply:
I just folded the average number of days into the text blocks. Otherwise, it looked weird.

Reply:
Now the whole thing looks like a block quote? Does it look fine when you build the docs locally? https://github.com/hail-is/hail/blob/091a979e9a2fea8129007ca87f6e533a5dbe73a8/hail/python/hailtop/batch/docs/service.rst

Reply:
<img width="564" alt="Screen Shot 2020-05-29 at 11 54 42 AM" src="https://user-images.githubusercontent.com/1693348/83279716-84ebab80-a1a3-11ea-9538-6a31fcd54cbe.png">